global.json

```diff
@@ -475,11 +475,11 @@
             "id": "wfm_mobile_demo_app",
             "name": "WFM Demo Mobile App",
             "type": "client_advanced_hybrid",
-            "repoUrl": "git://github.com/feedhenry-raincatcher/raincatcher-demo-mobile.git",
-            "githubUrl": "https://github.com/feedhenry-raincatcher/raincatcher-demo-mobile.git",
-            "repoBranch": "refs/heads/rhmap",
+            "repoUrl": "git://github.com/feedhenry-raincatcher/raincatcher-mobile.git",
+            "githubUrl": "https://github.com/feedhenry-raincatcher/raincatcher-mobile.git",
+            "repoBranch": "refs/heads/master",
             "icon": "icon-cordova",
-            "category": "Sample Apps",
+            "category": "WFM Apps",
```

Comment:
Is this okay to change? I think it affects platform UI adding a new category, so just wanted to make sure.